test/services/licenceSearchService.test.js

```diff
@@ -33,34 +39,106 @@ describe('licenceSearchService', () => {
         {
           bookingId: '1',
           prisonerNumber: 'AAAA11',
+          firstName: 'John',
+          lastName: 'Smith',
           prisonId: 'MDI',
           prisonName: 'Moorland (HMP & YOI)',
-          homeDetentionCurfewEligibilityDate: '01-01-2021',
+          homeDetentionCurfewEligibilityDate: moment().add(13, 'weeks').format('YYYY-MM-DD'),
         },
         {
           bookingId: '2',
           prisonerNumber: 'AAAA12',
+          firstName: 'Max',
+          lastName: 'Martin',
           prisonId: 'MDI',
           prisonName: 'Moorland (HMP & YOI)',
-          homeDetentionCurfewEligibilityDate: '01-01-2021',
+          homeDetentionCurfewEligibilityDate: moment().add(13, 'weeks').format('YYYY-MM-DD'),
         },
         {
           bookingId: '3',
           prisonerNumber: 'AAAA13',
+          firstName: 'Tim',
+          lastName: 'North',
           prisonId: 'MDI',
           prisonName: 'Moorland (HMP & YOI)',
-          homeDetentionCurfewEligibilityDate: '01-01-2021',
+          homeDetentionCurfewEligibilityDate: moment().add(13, 'weeks').format('YYYY-MM-DD'),
```

Comment:
Think we need to be test closer to fence posts here.. 
Would be good to have cases to cover day before, exactly 14 weeks, day after.